records_mover/session.py

```diff
@@ -41,6 +42,33 @@ def _infer_scratch_s3_url(session_type: str) -> Optional[str]:
     if "SCRATCH_S3_URL" in os.environ:
         return os.environ["SCRATCH_S3_URL"]
 
+    # TODO: do I want this in 'app.ini'?
+    # TODO: where else to document this?
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker

Reply:
I'm kind of thinking we create a CONFIG.md describing (for now) how to configure the S3 bucket, including the content below, and point to it on github.com in the error message when we raise a NoTemporaryBucketConfiguration or CredsDoNotSupportS3Export.  Seem reasonable?